src/screens/EPub/views/EditEPubChapter/ChapterEditor/ChapterContent.test.js

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from '@testing-library/react';
+// TypeError: Cannot read properties of undefined (reading '__buildHTMLFromChapter')
+// (src/screens/EPub/controllers/file-builders/EPubParser.js:6:46)
+import { v4 as uuidv4 } from 'uuid';
+import ChapterContent from './ChapterContent';
+
+const test_uuid = '9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d';
+
+jest.mock('uuid', () => {
+  return {
+    v4: jest.fn(() => test_uuid),
+  };
+});
+
+
+describe('ChapterContent Component', () => {
+    const mockDispatch = jest.fn();
```

Comment:
If we aren't using these variables anywhere else, might be simpler to just have `jest.fn()` in props directly similar to your other test files, though this works fine too since the readability is quite clear in both cases.